### DISCUSSION TITLE
Core ml Models

### DISCUSSION THREAD

**Discussion by joaourbano** [ryouchinsa/Rectlabel-support#205] [Q&A]
Hi,

I am looking for an implementation of yolov5 that can be used in Apple HW and Core ML.

Can someone help me out?

Thanks

**Comment by ryouchinsa** [comment]
Thanks for writing the issue.

For the yolov5 models, you can download here, such as yolov5s.pt.
https://github.com/ultralytics/yolov5/releases/tag/v6.2

To convert the .pt model to .mlmodel model, run the command.
python export.py --weights yolov5s.pt --include coreml
https://github.com/ultralytics/yolov5/issues/251

Then, you can load the yolov5s.mlmodel on RectLabel, process your images for automatic labeling.
https://rectlabel.com/help#process_images_coreml

Please let us know your opinion.

**Reply by joaourbano** [reply]
Hi,

Thank you for your wonderful support. I have managed to successfully covert the model.

However I am having problems trying to convert the following pre-trained model into createml (mlmodel file):

https://github.com/shayanalibhatti/Retail-Store-Item-Detection-using-YOLOv5

Could you please help me out?

Thank you.

**Reply by ryouchinsa** [reply]
Thanks for writing the issue. 
If the last_yolov5s_results.pt could not be exported to the Core ML model, please ask the developer how to use the export.py in the yolov5 repository.

**Reply by ryouchinsa** [reply]
Read this comment. 
https://github.com/shayanalibhatti/Retail-Store-Item-Detection-using-YOLOv5/issues/7#issuecomment-1378771327

**Comment by ryouchinsa** [comment]
Currently If you need our support to fix this problem, please let us know.
